roles/chatbot/templates/chatbot.deployment.yaml.j2

```diff
@@ -268,8 +268,7 @@ spec:
       restartPolicy: Always
       volumes:
         - name: ansible-chatbot-storage
-          persistentVolumeClaim:
-            claimName: "{{ ansible_operator_meta.name }}-chatbot-pvc"
+          emptyDir: {}
```

Comment:
It may be nice to set the sizeLimit to avoid any surprise:

https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
`If no size is specified, memory-backed volumes are sized to node allocatable memory.`

Reply:
Good point, than you Goneri, fixed.

Reply:
In https://github.com/ansible/ansible-wisdom-ops/pull/1565 , I specified `sizeLimit: 1000Mi`.

The storage is used for storing embedding models and vector db.  Currently (on my local environment) they are:
```
Embedding model: 419M
Vector DB: 121M
```
Even if the vector DB becomes bigger in future releases, I think 1000M would be enough.

Reply:
Thank you @TamiTakamiya added the same value for the Operator. 👍